website/source/api/json-jobs.html.md

```diff
@@ -704,6 +707,7 @@ An example `Update` block:
         "MinHealthyTime": 15000000000,
         "HealthyDeadline": 180000000000,
         "AutoRevert": false,
+        "AutoPromote": false,
```

Comment:
Will let others do a more thorough review, but noting that code for handling hcl job spec parsing for the update stanza is missing? 

Should handle the new field here https://github.com/hashicorp/nomad/blob/875597729afc4daaa88e58339199515c3f0c8d33/jobspec/parse.go#L1581 and update the tests

Reply:
Should also update the diff test to add autopromote to the test case - https://github.com/hashicorp/nomad/blob/d5dd9c2c692f951356f81bcc74847441c7c19eea/nomad/structs/diff_test.go#L2078